service_worker.js

```diff
@@ -287,3 +300,11 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     sendResponse({tabId: sender.tab.id});
   }
 });
+
+// Listen for changes to noBadgeAnimation option
+function logStorageChange(changes, area) {
+  if (area === 'sync' && 'noBadgeAnimation' in changes) {
+    optionsNoBadgeAnimation = changes['noBadgeAnimation'].newValue;
```

Comment:
nit

```suggestion
    optionsNoBadgeAnimation = changes.noBadgeAnimation.newValue;
```

Reply:
Fixed in https://github.com/GoogleChrome/web-vitals-extension/commit/010d88bc15b6f8ff6e048abb51bbf00d74f94fcd